docs/use_cases/batch_to_streaming/part1.md

```diff
@@ -0,0 +1,608 @@
+        C_MKTSEGMENT  CHAR(10) NOT NULL,
+        C_COMMENT     VARCHAR(117) NOT NULL
+) WITH (
+ 'connectors' = '[{
+    "transport": {
+      "name": "delta_table_input",
+      "config": {
+        "uri": "s3://batchtofeldera/customer",
+        "aws_skip_signature": "true",
+        "aws_region": "ap-southeast-2",
+        "mode": "snapshot_and_follow"
+      }
+    }
+ }]'
+);
+
+CREATE TABLE SUPPLIER (
+        S_SUPPKEY     INTEGER NOT NULL,
+        S_NAME        CHAR(25) NOT NULL,
+        S_ADDRESS     VARCHAR(40) NOT NULL,
+        S_NATIONKEY   INTEGER NOT NULL,
+        S_PHONE       CHAR(15) NOT NULL,
+        S_ACCTBAL     DECIMAL(15,2) NOT NULL,
+        S_COMMENT     VARCHAR(101) NOT NULL
+) WITH (
+ 'connectors' = '[{
+    "transport": {
+      "name": "delta_table_input",
+      "config": {
+        "uri": "s3://batchtofeldera/supplier",
+        "aws_skip_signature": "true",
+        "aws_region": "ap-southeast-2",
+        "mode": "snapshot_and_follow"
+      }
+    }
+ }]'
+);
+
+CREATE TABLE PARTSUPP (
+        PS_PARTKEY     INTEGER NOT NULL,
+        PS_SUPPKEY     INTEGER NOT NULL,
+        PS_AVAILQTY    INTEGER NOT NULL,
+        PS_SUPPLYCOST  DECIMAL(15,2)  NOT NULL,
+        PS_COMMENT     VARCHAR(199) NOT NULL
+) WITH (
+ 'connectors' = '[{
+    "transport": {
+      "name": "delta_table_input",
+      "config": {
+        "uri": "s3://batchtofeldera/partsupp",
+        "aws_skip_signature": "true",
+        "aws_region": "ap-southeast-2",
+        "mode": "snapshot_and_follow"
+      }
+    }
+ }]'
+);
+
+CREATE TABLE NATION  (
+        N_NATIONKEY  INTEGER NOT NULL,
+        N_NAME       CHAR(25) NOT NULL,
+        N_REGIONKEY  INTEGER NOT NULL,
+        N_COMMENT    VARCHAR(152)
+) WITH (
+ 'connectors' = '[{
+    "transport": {
+      "name": "delta_table_input",
+      "config": {
+        "uri": "s3://batchtofeldera/nation",
+        "aws_skip_signature": "true",
+        "aws_region": "ap-southeast-2",
+        "mode": "snapshot_and_follow"
+      }
+    }
+ }]'
+);
+
+CREATE TABLE REGION  (
+        R_REGIONKEY  INTEGER NOT NULL,
+        R_NAME       CHAR(25) NOT NULL,
+        R_COMMENT    VARCHAR(152)
+) WITH (
+ 'connectors' = '[{
+    "transport": {
+      "name": "delta_table_input",
+      "config": {
+        "uri": "s3://batchtofeldera/region",
+        "aws_skip_signature": "true",
+        "aws_region": "ap-southeast-2",
+        "mode": "snapshot_and_follow"
+      }
+    }
+ }]'
+);
+```
+
+:::note
+Currently, Feldera does not support importing the schema from Delta Lake.
+However, this functionality is planned for future releases.
+:::
+
+## Queries
+
+The queries from the previous section can be used in Feldera without
+modification. However, we define these views as
+[materialized](https://docs.feldera.com/sql/materialized) so that we
+can use [ad-hoc](https://docs.feldera.com/sql/ad-hoc) queries to query them
+later. These views do not need to be **materialized** if you do not intend to
```

Comment:
you should probably explain the difference. if the views are not materialized the user has to collect the data outside Feldera.